docs/sources/tempo/configuration/_index.md

```diff
@@ -330,7 +339,7 @@ metrics_generator:
         span_metrics:
 
             # Buckets for the latency histogram in seconds.
-            [histogram_buckets: <list of float> | default = 0.002, 0.004, 0.008, 0.016, 0.032, 0.064, 0.128, 0.256, 0.512, 1.02, 2.05, 4.10]
+            [histogram_buckets: <list of float> | default = 0.002, 0.004, 0.008, 0.016, 0.032, 0.064, 0.128, 0.256, 0.512, 1.02, 2.05, 4.10, 8.20, 16,40]
```

Comment:
Is this correct?
https://github.com/grafana/tempo/blob/251bf5a9a74fd586ffd3dedcc2c8000f36aa9a78/modules/generator/processor/spanmetrics/config.go#L54

Should be 14 entries correct?